.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+    root: true,
+    env: {
+        browser: true,
+        node: true
+    },
+    parserOptions: {
+        parser: 'babel-eslint'
+    },
+    extends: [
+        //'@nuxtjs',
+        //'plugin:nuxt/recommended',
+        //'plugin:vue/recommended',
```

Comment:
We can remove the comment if not needed.

Reply:
Resolved